modules/installation-creating-azure-dns.adoc

```diff
@@ -65,20 +69,34 @@ $ az deployment group create -g ${RESOURCE_GROUP} \
   --parameters baseName="${INFRA_ID}"<1>
 ----
 <1> The base name to be used in resource names; this is usually the cluster's infrastructure ID.
+endif::ash[]
 
-. Create an `api` and `api-int` DNS record in the public zone for the API public load
+ifdef::azure[]
+. Create an `api` DNS record in the public zone for the API public load
 balancer. The `${BASE_DOMAIN_RESOURCE_GROUP}` variable must point to the
 resource group where the public DNS zone exists.
+endif::azure[]
+
+ifdef::ash[]
+. Create an `api` DNS record and an `api-int` DNS record. When creating the API DNS records, the `${BASE_DOMAIN_RESOURCE_GROUP}` variable must point to the resource group where the DNS zone exists.
 endif::ash[]
 
 .. Export the following variable:
 +
 [source,terminal]
 ----
-$ export PUBLIC_IP=`az network public-ip list -g ${RESOURCE_GROUP} --query "[?name=='${INFRA_ID}-master-pip'] | [0].ipAddress" -o tsv`
+$ export PUBLIC_IP=$(az network public-ip list -g ${RESOURCE_GROUP} --query "[?name=='${INFRA_ID}-master-pip'] | [0].ipAddress" -o tsv)
```

Comment:
```suggestion
$ export PUBLIC_IP=`az network public-ip list -g ${RESOURCE_GROUP} --query "[?name=='${INFRA_ID}-master-pip'] | [0].ipAddress" -o tsv`
```
For consistency with the other variable export commands.

Reply:
Agreed. Done.